OMEdit/OMEdit/OMEditGUI/Annotations/LineAnnotation.cpp

```diff
@@ -1315,6 +1315,19 @@ void LineAnnotation::duplicate()
   pLineAnnotation->setSelected(true);
 }
 
+ void LineAnnotation::redraw(QString const& annotation, std::function<void()> updateAnnotationFunction)
```

Comment:
Isn't the normal style `const QString& annotation`. I don't know what are the implications of both.

Reply:
You are right, C++ being C++ the two things are actually the same if I am not mistaken. Will  change and update again